pkg/aws_config_client/completer_test.go

```diff
@@ -238,7 +287,7 @@ func TestCalculateDefaultProfileName(t *testing.T) {
 
 	r := require.New(t)
 
-	c := NewCompleter(nil, generateDummyData())
+	c := NewCompleter(nil, generateDummyData(), "", "")
```

Comment:
@ryanking do you have the time & bandwidth to add a test that includes a nonempty region and/or role? If not, I'll see if I can add it later today, test it, & approve it.

Reply:
I already added one [here](https://github.com/chanzuckerberg/aws-oidc/pull/167/files#diff-8e5600bde6250e3fa534164848d1cb07955ba52a1e4c1fb602fb3bd10c851b1eR205), which has non-empty values for both of those parameters.